### PR TITLE
feat(source): adding the source page

### PR DESCRIPTION
**Pull request by boaz0**
- [x] Work on the "remove source" dialog message
- [x] work on the "remove source" API - client side
- [x] Expand table item
- [x] Filter by name/type
- [x] Locales support
- [x] Selectable table item (commented)
- [x] Add states: no match found, Loading, Error and Empty

### CODE REVIEW

**Comment by dlabrecq** [comment]
Please let me know when the PR is ready for a review. Unless you need some feedback now?

**Comment by boaz0** [comment]
I'm blocked now. I keep getting 404 when attempting to fetch sources or reports.

**Comment by dlabrecq** [comment]
Try rebasing your code. Chris made some updates to the UI to accommodate Insights recent changes.

You need to point your browser at a new URL. See https://github.com/project-koku/koku-ui/pull/622

**Comment by boaz0** [comment]
//cc @dlabrecq waiting for your review. Thanks.

File: src/routes.tsx [hunk 1]
```diff
@@ -57,6 +61,13 @@ const routes: AppRoute[] = [
     exact: true,
     icon: MoneyBillIcon,
   },
+  {
+    path: '/sources',
+    labelKey: 'navigation.source_details',
+    component: SourceDetails,
+    exact: true,
+    icon: MoneyBillIcon,
+  },
```
[the remark this answers is not in the document]
Comment (boaz0) [reply src/routes.tsx]: Nope, it's not working :sob:

**Comment by boaz0** [comment]
@dlabrecq - done. renamed the files/directories and changed the "Add Source" button in the Overview page to "Add Sources" that will go to the Source Setting page.

**Comment by boaz0** [comment]
Thanks a lot @dlabrecq

**Pull request merged**